crates/miden-objects/src/block/block_account_update.rs

```diff
@@ -46,7 +46,7 @@ impl BlockAccountUpdate {
         self.final_state_commitment
     }
 
-    /// Returns the description of the updates for on-chain accounts.
+    /// Returns the description of the updates for accounts whose state is public on chain.
```

Comment:
nit: AFAICS this method can also be called for private accounts, and then the `AccountUpdateDetails` variant is `Private`. So technically this comment is (and also was before) incomplete